linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_name_format/ansible/shared.yml

```diff
@@ -10,10 +10,32 @@
   {{%- set auditd_conf_path=audisp_conf_path + "/auditd.conf" %}}
 {{%- endif %}}
 
-{{{ ansible_set_config_file(file=auditd_conf_path,
-                  parameter="name_format",
-                  value="hostname",
-                  create=true,
-                  separator=" = ",
-                  separator_regex="\s*=\s*",
-                  prefix_regex="(?i)^\s*") }}}
+{{{ ansible_instantiate_variables("var_auditd_name_format") }}}
+
+- name: Set type of computer node name logging in audit logs
```

Comment:
Please, for tasks names use the prefix defined in the Project Style Guide:
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible